pkg/email/template/template.go

```diff
@@ -0,0 +1,34 @@
+package template
+
+import (
+	"strings"
+	"text/template"
+)
+
+const (
+	ResetPasswordPath  = "/auth/resetpassword?token="
+	ResetEmailTemplate = `Hello, <br>
+	A request has been received to change the password to your account,  <br>
+	Click on this link to reset your password <br>
+	{{.Host}}{{.ResetPasswordPath}}{{.Tkn}} <br>
+	If you did not initiate this request you can please diregard this email. <br>
+	Thanks  <br>
+	The LakeFS team <br>
+	`
```

Comment:
All the text is indent - everything should be without spaces/tabs on the left.
It means that you are using HTML and you should use the html template to render the content.